cmd/init.go

```diff
@@ -86,7 +86,7 @@ func initialize(ctx context.Context, providers []string) error {
 			},
 		},
 	}
-	if diags := config.ValidateCQBlock(&mainConfig.CloudQuery); diags.HasErrors() {
```

Comment:
`ValidateCQBlock` did more stuff that validation (for example, setting the providers and policies download directories). This is done as a part of `HandleDecodedConfig` now